src/components/themed/Alert.tsx

```diff
@@ -96,11 +96,11 @@ export function Alert({
         {icon}
         <EdgeText style={styles.title}>{title}</EdgeText>
       </View>
-      {message && (
+      {message !== null ? (
```

Comment:
Don't use `!==` for `null`. Because `undefined` is also a valid message, but `undefined` is not exactly the same as `null`.